static/app/components/organizations/projectPageFilter/index.tsx

```diff
@@ -54,6 +54,11 @@ export interface ProjectPageFilterProps
    * Message to show in the menu footer
    */
   footerMessage?: React.ReactNode;
+  /**
+   * Override the selected projects that are displayed.
+   * Does NOT override the projects in the store.
+   */
+  projectOverride?: number[];
```

Comment:
I think you should give a bit more context in this comment, that this is only meant to show a disabled state for all projects not that we're allowing an override when a user can actually use it